project-reports-lib.ts

```diff
@@ -8,7 +8,43 @@ export * from './project-reports-schemes';
 // filter cards by label
 //
 export function filterByLabel(cards: IssueCard[], name: string): IssueCard[] {
-    return cards.filter((card) => card.labels.findIndex(label => label.name === name) >= 0);
+    return cards.filter((card) => card.labels.findIndex(label => label.name.toLowerCase() === name.toLowerCase()) >= 0);
+}
+
+//
+// Calculate cycle time for a card
+// The time, in days, a unit of work spends between the first day it is actively being worked on until the day it is closed.
+// In this case, since a project card has events, we look for the event that moved or added a card to the "Accepted" column
+// and subtract it from the time that the card moved to the `Done` column.
+//
+export function calculateCycleTime(card: IssueCard): number {
+    // console.log(`Calculating cycle time for [${card.title}]`);
+
+    // cycle time starts at Accepted, ends at Done.
+    let accepted_time:Date = null;
+    let done_time:Date = null;
+    card.events.forEach((event)=> {
```

Comment:
the engine already provides this data: https://github.com/bryanmacfarlane/project-reports-action/blob/master/interfaces.ts#L116

It's also not so straight forward because you have to handle moving backwards (not counting).   explained in the comments above.  That is processed onto the card before your report is called.

Reply:
also see here: https://github.com/bryanmacfarlane/project-reports-action/blob/master/util.ts#L80